activejob/test/support/integration/dummy_app_template.rb

```diff
@@ -14,7 +14,7 @@ class TestJob < ActiveJob::Base
 
   def perform(x)
     File.open(Rails.root.join("tmp/\#{x}"), "w+") do |f|
-      f.write x
+      f.write I18n.locale
```

Comment:
@kaspth I added the integration test and used the fact that the content of the file wasn't asserted before. I thought that might be slightly better than adding a whole new job for this one test.